pkg/util/util.go

```diff
@@ -234,61 +244,84 @@ func ParseClusterYaml(file string) (*clusterv1.Cluster, error) {
 }
 
 // ParseMachinesYaml extracts machine objects from a file.
-func ParseMachinesYaml(file string) ([]*clusterv1.Machine, error) {
+func ParseMachinesYaml(file string) ([]*clusterv1.Machine, []*clusterv1.MachineDeployment, error) {
```

Comment:
I think a dedicated `ParseMachineDeploymentsYaml` can be defined here.
Someone might want to add a `ParseMachineSetsYaml` in the future.

Reply:
on second thought - the "Machines" in the name ParseMachinesYaml refers to the `--machines` flag. Are you suggesting that we have a `--machineDeployments`?

Reply:
I don't see such a tight relationship between the flag and the function. I was thinking about supporting control planes defined as `machineDeployments`. I think the `--machines` flag should support both `machines` and `machineDeployments` so no need for an extra flag.

When the `--machines` flag is set the code should execute both functions `ParseMachinesYaml` and `ParseMachineDeploymentsYaml` and then filter out the control planes. Just some thoughts, I am new to all this :)

Reply:
I actually agree with that suggestion. That's what I was thinking. Just wanted to make sure I was understanding your comment correctly. :)